locales/en-US/tools.ftl

```diff
@@ -154,7 +154,7 @@ install-other-methods-link = Learn more
 ## components/tools/rustup.hbs
 
 tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. See <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">"Other Installation Methods"</a> if you are on Windows.
-tools-rustup-windows = It looks like you’re running Windows. To start using Rust, download the installer, then run the program and follow the onscreen instructions. See <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">"Other Installation Methods"</a> if you are not on Windows.
+tools-rustup-windows = It looks like you’re running Windows. To start using Rust, download the installer, then run the program and follow the onscreen instructions. You may need to install the <a href="https://visualstudio.microsoft.com/visual-cpp-build-tools/">Visual Studio C++ Build tools</a> when prompted to do so. If you are not on Windows see <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">"Other Installation Methods"</a>.
```

Comment:
Would you be able to change the key here and in the html to `tools-rustup-windows-2`? This will help by letting our translators know that this content needs to be updated.

Reply:
@XAMPPRocky just for my own interest, how does changing that key allow them to know? Cheers

Reply:
@Fasani Sure! We use a localisation service called pontoon <https://pontoon.rust-lang.org> and changing the key will create a new untranslated key for every other locale. So pontoon will show them they haven’t translated the new text.

Reply:
Ok, I think that's fixed now?